project-words.txt

```diff
@@ -1,19 +1,96 @@
+acqktp
+AESSBOX
+allclose
+arange
+argmax
+argmin
+argsort
+ascad
 astype
+autoreload
+batchnorm
+batchnormalization
+bibtex
+booktitle
+Bursztein
+burszteindc
+burzteindc
 bytelist
 chipwhisperer
+chipwispher
```

Comment:
This one seems like a typo somewhere in the doc/code rather than a word we should add.

Reply:
Agree, but I wanted to be careful with fixing typos that change API during this PR (it is large and otherwise full of rather save typo fixes). See issue https://github.com/google/scaaml/issues/90

Part of function name at https://github.com/google/scaaml/blob/master/scaaml/utils.py#L99

That being said I am happy to fix it in this PR, but I thought it would be easier to review this separately.